pkgs/tools/z/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, lib, ... }:
+
+stdenv.mkDerivation rec {
+  name = "z-${version}";
+  version = "1.11";
+
+  src = fetchFromGitHub {
+    owner = "rupa";
+    repo = "z";
+    rev = "v${version}";
+    sha256 = "13zbgkj6y0qhvn5jpkrqbd4jjxjr789k228iwma5hjfh1nx7ghyb";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  installPhase = ''
+    install -Dm 644 z.sh $out/share/z.sh
+    install -Dm 644 z.1 $out/share/man/man1/z.1
+  '';
+
+  meta = {
```

Comment:
```
meta = with stdenv.lib; {
```

is commonly used, then you can drop `lib.` from license and maintainers below.

Also `homepage = https://github.com/rupa/z;` would be nice to add :)